Sources/PackageGraph/PackageGraph+Loading.swift

```diff
@@ -687,11 +689,11 @@ private class ModuleAliasTracker {
     init() {}
 
     func addAlias(_ alias: String,
-                  of targetName: String,
-                  for product: String,
-                  from originPackage: PackageIdentity,
-                  in parentPackage: PackageIdentity) {
-        let model = ModuleAliasModel(name: targetName, alias: alias, originPackage: originPackage, parentPackage: parentPackage)
+                  target: String,
+                  product: String,
+                  originPackage: PackageIdentity,
```

Comment:
is originPackage the package from which the target / product is from?

Reply:
yes